packages/playwright-core/src/server/registry/index.ts

```diff
@@ -37,6 +37,10 @@ const PACKAGE_PATH = path.join(__dirname, '..', '..', '..');
 const BIN_PATH = path.join(__dirname, '..', '..', '..', 'bin');
 
 const PLAYWRIGHT_CDN_MIRRORS = [
+  'https://playwright.azureedge.net/dbazure/download/playwright', // ESRP
+  'https://playwright.download.prss.microsoft.com/dbazure/download/playwright', // ESRP Fallback
+
+  // Old CDNs, to be removed.
   'https://playwright.azureedge.net',
   'https://playwright-akamai.azureedge.net',
```

Comment:
Can you also add a comment here that akamai and verizon are actually pointing to a different falvor of azure cdn and will soon become same?